package.json

```diff
@@ -28,28 +28,39 @@
     "google-gax": "^1.7.5"
   },
   "devDependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.0.3",
     "c8": "^6.0.0",
     "chai": "^4.2.0",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-prettier": "^3.0.1",
+    "gts": "^1.1.0",
     "jsdoc": "^3.6.2",
     "jsdoc-fresh": "^1.0.1",
     "jsdoc-region-tag": "^1.0.2",
     "linkinator": "^1.4.0",
     "mocha": "^6.1.4",
-    "prettier": "^1.17.0"
+    "pack-n-play": "^1.0.0-2",
+    "prettier": "^1.17.0",
+    "typescript": "~3.5.0"
```

Comment:
This should go straight to `3.7.0`.  Do we have renovate running on `gapic-generator-typescript`?

Reply:
Huh. This is an existing repo so `package.json` is not coming from the generator. I just did `npx gts init` here and turns out it brings some outdated stuff. Fixing now